django/contrib/admin/tests.py

```diff
@@ -14,11 +15,41 @@ def process_response(self, request, response):
         return response
 
 
+class AdminSeleniumMetaClass(type):
+    def __new__(cls, name, bases, attrs):
+        """
+        Dynamically inject new methods for running tests in different browsers
+        when multiple browser specs are provided (e.g. --selenium=firefox,chrome).
+        """
+        all_specs = os.environ.get('DJANGO_SELENIUM_SPECS', '').split(',')
+        if all_specs:
+            for key, value in attrs.items():
+                if isinstance(value, types.FunctionType) and key.startswith('test'):
+                    func = value
+                    for index, spec in enumerate(all_specs):
+                        new_func_name = '%s__%s' % (key, spec.replace('/', '_'))
```

Comment:
This class is not doing much right now, I thought we wanted to keep it for building upon it in future. As a matter of fact, removing it completely won't break/change anything.
It was supposed to work when more than one browser were passed to the `--selenium` flag which as I mentioned earlier is not working.